### ISSUE TITLE
Seems like zend.com php7 repo is failing

### ISSUE THREAD

**Issue by raptor235** [wpengine/hgv#302]
==> sites: Failed to fetch http://repos.zend.com/zend-server/early-access/php7/repos/ubuntu/Packages  404  Not Found [IP: 174.35.50.71 80]

https://www.dropbox.com/s/mnrvwe4q1t6xk3o/Screenshot%202016-03-22%2014.12.37.png?dl=0

There should be a check for this and skip over the package if it's not available

**Comment by qnhuly** [comment]
I'm hitting the same issue.  Does that mean I need to install php7 separately?

**Comment by willenglishiv** [comment]
same here

**Comment by qnhuly** [comment]
go to /hgv/provisioning/roles/php-fpm/tasks/main.yml and comment this last line out:
#- include: php7.yml

Then:
Change into the directory hgv.
Run vagrant destroy.
Run vagrant up

That will fix the above error.  It caches in the local vagrant so running "vagrant provision" won't work, you have to delete it and re-create from scratch.

**Comment by markkelnar** [comment]
Thank you @raptor235 and @qnhuly for the details.
I'm looking into a few options.  

(a) is to provide a way to ignore php7 by default.
(b) is to work on using a standard Ubuntu distribution of the PHP7 package.
(c) is to use a recent working version of the php7-nightly build that I can find and run with that until we can do (b) above.

**Comment by markkelnar** [comment]
I've proposed a solution for the approach described for (c).  I put the debian package in a WPE Labs controlled AWS S3 bucket and the provisioning can download it from there and install it if php7-nightly is not already installed in the vagrant.

If this seems sound (cause I've tested it and it works as expected), we can proceed with merging it to master.

**Comment by raptor235** [comment]
Hey @markkelnar , just posting this here as well so it's accessible online... the issue still exists with the new pull request.

`[ home/hgv ] - $ vagrant provision
==> hgv: Running provisioner: fix-no-tty (shell)...
    hgv: Running: inline script
==> hgv: Running provisioner: shell...
    hgv: Running: /var/folders/2f/8rl_2_ws4491sqqn233vvz1r0000gn/T/vagrant-shell20160323-11271-l5iswj.sh
==> hgv:  -------------------------  -----------------
==> hgv: |   _    _                ||  __      __     |
==> hgv: |  | |  | |               ||  \ \    / /     |
==> hgv: |  | |__| |  __           ||   \ \  / /      |
==> hgv: |  |  __  |/ _  \         ||    \ \/ /       |
==> hgv: |  | |  | | (_| |         ||     \  /        |
==> hgv: |  |_|  |_|\__, |         ||      \/         |
==> hgv: |           __/ |___  __  ||        ___ ____ |
==> hgv: |          |___/( _ )/  \ ||       |_  )__ / |
==> hgv: |               / _ \ () |||        / / |_ \ |
==> hgv: |               \___/\__/ ||       /___|___/ |
==> hgv:  -------------------------  -----------------
==> hgv: Updating APT sources.
==> hgv: W
==> hgv: : Failed to fetch http://repos.zend.com/zend-server/early-access/php7/repos/ubuntu/Packages  404  Not Found [IP: 174.35.56.207 80]
==> hgv:
==> hgv: E: Some index files failed to download. They have been ignored, or old ones used instead.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.`

**Comment by raptor235** [comment]
Do I need to run vagrant destroy before this yml gets read in?

**Comment by raptor235** [comment]
Vagrant destroy did the trick :)

**Comment by jb510** [comment]
Same.  Patch worked, but only after vagrant destroy :( Really need to make DBs persistent at some point.

**Comment by storypixel** [comment]
Same problem and now I'm trying to restore my old db from a backup which is a whole other issue.

**Comment by programmin1** [comment]
PHP7 is now the latest stable, and soon will be the second-latest release. That's probably why it's failing on that repos.zend.com/zend-server/**early-access**/php7/repos/ubuntu/Packages url.

I saw the same on my Ubuntu 14.04 machine.
